From: Michael Fladischer <FladischerMichael@fladi.at>
Date: Wed, 22 Nov 2023 16:39:26 +0000
Subject: Fix duplicate entries in tox.ini.

---
 tox.ini | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/tox.ini b/tox.ini
index 352f489..46056d4 100644
--- a/tox.ini
+++ b/tox.ini
@@ -8,7 +8,8 @@ envlist =
     pypy39-django{32,40,41,main},
 
 [testenv]
-allowlist_externals = ./run.sh
+allowlist_externals =
+    ./run.sh
 deps =
     django32: Django>=3.2,<3.3
     django40: Django>=4.0,<4.1
@@ -18,10 +19,6 @@ deps =
     pymemcache>=4.0,<5.0
     django-redis>=5.2,<6.0
     flake8
-
-allowlist_externals =
-    */run.sh
-
 commands =
     ./run.sh test {posargs}
     ./run.sh flake8
